src/monitor/monitor.c

```diff
@@ -231,10 +231,10 @@ static int client_registration(struct sbus_request *dbus_req, void *data)
     DBusError dbus_error;
     dbus_uint16_t svc_ver;
     dbus_uint16_t svc_type;
+    dbus_uint16_t explicitly_configured = true;
     char *svc_name;
     dbus_bool_t dbret;
     int ret;
-    bool explicitly_configured = true;
 
```

Comment:
I do think so, but I'll re-try it just to be sure my memory isn't tricking me.
May be the case I was trying to use the ordinary "bool" instead of the "dbus_bool_t".